src/components/Editor/QuillEditor.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useRef, useState } from "react";
+import "../../css/quillEditor.css";
+import { useFirebase } from "react-redux-firebase";
+import { useDispatch, useSelector } from "react-redux";
+import { Prompt } from "react-router-dom";
+import { setCurrentStep } from "../../store/actions";
+import * as Y from "yjs";
+import { QuillBinding } from "y-quill";
+import Quill from "quill";
+import QuillCursors from "quill-cursors";
+import { FirestoreProvider, getColor } from "@gmcfall/yjs-firestore-provider";
+
+import { onlineFirebaseApp } from "../../config";
```

Comment:
Can you please tell me why we have left this empty line?

Reply:
Thank you for pointing out the empty line that was left behind. I'll clear that.

Reply:
Cool!